apps/examples/simple-example/config/app-config.json

```diff
@@ -73,6 +73,9 @@
     "download_something": {
       "type": "GET"
     },
+    "download_something_streamed": {
+      "type": "GET"
```

Comment:
[Suggestion] use auth Unsecured to allow testing this with wget or a browser directly.